tests/ottest/fixtures/blockchain/Restart.cpp

```diff
@@ -0,0 +1,264 @@
+// Copyright (c) 2010-2022 The Open-Transactions developers
+// // This Source Code Form is subject to the terms of the Mozilla Public
+// // License, v. 2.0. If a copy of the MPL was not distributed with this
+// // file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ottest/fixtures/blockchain/RegtestSimple.hpp"  // IWYU pragma: associated
+
+#include <gtest/gtest.h>
+#include <opentxs/opentxs.hpp>
+#include <ostream>
+#include <string_view>
+#include <tuple>
+
+#include "ottest/fixtures/blockchain/Restart.hpp"
+#include "ottest/fixtures/common/User.hpp"
+
+namespace ottest
+{
+const User& Restart_fixture::createUser(
+    const std::string& name,
+    const std::string& words)
+{
+    auto [user_alice, success_alice] =
+        CreateClient(opentxs::Options{}, instance++, name, words, address_);
+    EXPECT_TRUE(success_alice);
+
+    return user_alice;
+}
+
+const User& Restart_fixture::createSenderAlice()
+{
+    return createUser(name_alice, words_alice);
+}
+
+const User& Restart_fixture::createReceiverBob()
+{
+    return createUser(name_bob, words_bob);
+}
+
+void Restart_fixture::mineForBothUsers(
+    const User& user_bob,
+    const User& user_alice)
+{
+    Height begin = targetHeight;
+    auto scan_listener_alice = std::make_unique<ScanListener>(*user_alice.api_);
+    auto scan_listener_bob = std::make_unique<ScanListener>(*user_bob.api_);
+
+    targetHeight += blocks_number_;
```

Comment:
the meaning/use of `blocks_number_` is blurry in the test. it looks like you are using it as a constant increment in some places, as a reference value in others. 
I read this property as `currentHeight` of the blockchain with corresponding `targetHeight` as the target (so based on the two values we can actually calculate how many blocks should be mined.